tls.go

```diff
@@ -43,9 +43,8 @@ func initTLS(cfg *Config) (*tls.Config, error) {
 	}
 
 	return &tls.Config{
-		ServerName:           cfg.TLS.ServerName,
-		MinVersion:           tls.VersionTLS12,
-		GetClientCertificate: getClientCertificate(cfg),
```

Comment:
Unfortunately, we can't remove `getClientCertificate`. It is used to hot-reload certificates.